server/activityItem/__tests__/fetch.test.ts

```diff
@@ -1,4 +1,3 @@
-/* global describe, it, expect, beforeEach, beforeAll, afterAll */
```

Comment:
I also made a small tweak to our eslint config so we don't have to define jest globals in our test files anymore.